reflex-app/src/animations/Hamburger/index.js

```diff
@@ -16,6 +20,8 @@ class Hamburger extends PureComponent {
    this.burgerRef = React.createRef();
    this.animation = null;
    this.state = {isBurger: false};
+   this.color = '';
```

Comment:
I don't think these are necessary seeing that you've already got default values in line 13 and 8

Reply:
Removed them!